src/app/(proper_react)/redesign/(authenticated)/user/welcome/remove/page.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getL10n } from "../../../../../../functions/server/l10n";
+import { getOnerepProfileId } from "../../../../../../../db/tables/subscribers";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../api/utils/auth";
+import { NextResponse } from "next/server";
+import { optoutProfile } from "../../../../../../../external/onerep";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Results", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserWelcomeRemove() {
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber?.id) {
+    throw new Error("No session");
+  }
+
+  // Fetch list of subscriptions.
+  let subscriptions;
+  const bearerToken = session?.user.subscriber?.fxa_access_token;
+  if (bearerToken) {
+    const result = await fetch(
+      `${process.env.OAUTH_API_URI}/oauth/mozilla-subscriptions/customer/billing-and-subscriptions`,
+      {
+        headers: {
+          "Content-Type": "application/json",
+          authorization: `Bearer ${bearerToken}`,
+        },
+      }
+    );
+    if (result.ok) {
+      subscriptions = await result.json();
+    }
+  } else {
+    console.error("User has no bearer token");
+  }
+
+  type FxaSubscriptionResponse = {
```

Comment:
tiniest of nits: In other places we call it "FxA"

https://github.com/mozilla/blurts-server/blob/d67b9acd84c6923114327c76408687fdf8ab1730/src/db/tables/subscribers.js#L135

Although looks like we use `FxA` about 5:3 over `Fxa`, so&hellip;

<details>
<summary><kbd>git grep -Pn "(const|let|var|function) [a-zA-Z]+FxA[A-Z][a-zA-Z]+" | wc -l # 5</kbd></summary>

```sh
src/app/api/v1/fxa-rp-events/route.ts:186:        const currentFxAProfile = subscriber?.fxa_profile_json || {};
src/controllers/fxaRpEvents.js:151:        const currentFxAProfile = subscriber?.fxa_profile_json || {}
src/db/tables/subscribers.js:135:async function updateFxAData (subscriber, fxaAccessToken, fxaRefreshToken, fxaProfileData) {
src/db/tables/subscribers.js:163:async function updateFxAProfileData (subscriber, fxaProfileData) {
src/db/tables/subscribers.js:180:async function removeFxAData (subscriber) {
```
</details>


<details>
<summary><kbd>git grep -Pn "(const|let|var|function) [a-zA-Z]+Fxa[A-Z][a-zA-Z]+" | wc -l # 3</kbd></summary>

```sh
src/app/api/v1/fxa-rp-events/route.ts:60:const authenticateFxaJWT = async (req: NextRequest) => {
src/controllers/fxaRpEvents.js:47:const authenticateFxaJWT = async (req) => {
src/db/tables/subscribers.js:59:async function getSubscriberByFxaUid (uid) {
```
</details>